exercises/raindrops/instructions.md

```diff
@@ -0,0 +1,24 @@
+# Instructions
+
+Your task is to convert a number into its corresponding raindrop sounds.
+
+If a given number:
+
+- is divisible by 3, add 'Pling' to the result.
+- is divisible by 5, add 'Plang' to the result.
+- is divisible by 7, add 'Plong' to the result.
+- **is not** divisible by 3, 5, or 7, the result should be the digits of the number.
+
+## Examples
+
+- 28 is divisible by 7, but not 3 or 5, so the result would be `"Plong"`.
+- 30 is divisible by 3 and 5, but not 7, so the result would be `"PlingPlang"`.
+- 34 is not divisible by 3, 5, or 7, so the result would be `"34"`.
+
+~~~~exercism/note
+The simplest way to test if one number is evenly divisible by another is to use the [remainder][remainder] or [modulo][modulo] operator.
+
+[remainder]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Remainder
```

Comment:
You are reffering to javascript here, even if javascript isnt the only language with a remainder operator.

Reply:
I am, but I think it's a good doc on what the remainder operator is. I'm welcome to alternative links if they're not overly complex (e.g. wikipedia!)

Reply:
Also, FWIW, most languages have remainder - it's actually modulo that's the rarity.

Reply:
I do feel that linking to JS docs is a bit odd, but the contents is good. I couldn't find a better page, except for maybe the wiki page.

Reply:
Another option would be to explain the remainder here

Reply:
The best solution will be to have a doc on Exercism that we link to. Let's do that (as it's always part of the plan and this is a low-hanging fruit), but let's link this to the JS doc for now until we have that, then update when we've added that doc.

Reply:
Actually, I'll do it now before we merge.

Reply:
@ErikSchierboom Updated.